apps/judicial-system/web/src/routes/Shared/IndictmentOverview/IndictmentOverview.tsx

```diff
@@ -247,6 +249,23 @@ const IndictmentOverview: FC = () => {
               <InfoCardActiveIndictment displayVerdictViewDate />
             )}
           </Box>
+          {workingCase.state === CaseState.COMPLETED &&
```

Comment:
NIT: This check + the component below is repeated several times in the PR. Consider extracting a helper to check the condition for a given case and maybe have a specific conclusion component.